### DISCUSSION TITLE
New Venue Type suggestion for Motorway Service Area DOOH

### DISCUSSION THREAD

**Discussion by NickMawditt** [openooh/venue-taxonomy#101] [Ideas] [answered]
We suggest the creation of a new venue type on the basis that the current framework doesn't encapsulate the true scale of digital OOH opportunity within the Motorway Service Area (MSA), by excluding motorways from road classification. It currently sits in Billboards, which is a very broad category; the MSA environment provides advertisers a solus opportunity for consumer eyeballs, each its own unique space where people stop and engage with digital screens. Therefore, as i-media, we propose reclassifying the environment and incorporating the unregistered motorway network within new categories called NATIONAL MAJOR ROAD NETWORK and NATIONAL TRANSPORT NETWORK.

**Comment by jayshao** [comment, marked as answer]
I suspect in the current taxonomy this would be expressed as grandchildren under `301 - Billboards` - e.g. perhaps split Billboards based on type of road?

**Reply by jayshao** [reply]
Added an issue under https://github.com/openooh/venue-taxonomy/issues/109 for consideration
